sources/vcdMerge/src/SignalSource.cpp

```diff
@@ -97,11 +99,28 @@ void MERGE::SignalSource::Create()
 
 void MERGE::SignalSource::SetFormat(const std::string &rFormat)
 {
-    if (rFormat != "T")
+    if (rFormat == "T")
     {
-        throw EXCEPTION::VcdException(EXCEPTION::Error::INVALID_LOG_FILE_FORMAT,
-                                      "Invalid log file format: " + rFormat);
+        m_pSignalFactory = std::make_unique<PARSER::SignalFactory>();
+        return;
     }
+
+    if (rFormat.length() > 3)
+    {
+        if ((rFormat[0] == 'U') &&
+            (rFormat[1] == '{') &&
+            (rFormat[rFormat.length() - 1] == '}'))
+        {
+            const std::string filename(rFormat.substr(2, rFormat.length() - 3));
+            IsAccessible(filename);
```

Comment:
What?
This should read: `if(!IsAccessible(filename)) { error; }` In current form this is unreadable.

Reply:
How about renaming the method to 'VerifyAccessibility()'? The method throws an exception if the file is not available. It is used in other places as well. That's why I wanted to avoid 'if () {throw exception();}' duplicates.

Reply:
This is still bad for me. Who know what that function does? It would have to be named `VerifyFileAccesibleAndThrowIfNot()`, that doesn't look good. I wouldn't wory about duplicate `if, throw`, this clearly shows logic. Every `if` body could also be separate function for even greater clarity.